packages/tree-widget/src/components/trees/CategoriesTree.tsx

```diff
@@ -53,6 +58,40 @@ export function CategoriesTreeComponent(props: CategoriesTreeComponentProps) {
     );
   }, [props.iModel, filteredProvider]);
 
+  const invert = React.useCallback(async () => {
+    const activeView = IModelApp.viewManager.getFirstOpenView();
+    const ids = await getCategories(props.iModel, activeView, filteredProvider);
+    if (!activeView) {
```

Comment:
wouldn't you want to check if `activeView` is defined prior to calling `getCategories`?

Reply:
That call takes an undefined viewport, but yes, in this case it makes sense to do it after. I'll fix it. Thx

Reply:
Fixed.